prefab_cloud_python/client.py

```diff
@@ -67,40 +78,42 @@ def get(self, key, default="NO_DEFAULT_PROVIDED", context="NO_CONTEXT_PROVIDED")
                 key, default=default, context=self.resolve_context_argument(context)
             )
 
-    def enabled(self, feature_name, context="NO_CONTEXT_PROVIDED"):
+    def enabled(
+        self, feature_name: str, context: str | Context = "NO_CONTEXT_PROVIDED"
+    ) -> bool:
         return self.feature_flag_client().feature_is_on_for(
             feature_name, context=self.resolve_context_argument(context)
         )
 
-    def is_ff(self, key):
+    def is_ff(self, key: str) -> bool:
         raw = self.config_client().config_resolver.raw(key)
         if raw is not None and raw.config_type == Prefab.ConfigType.Value(
             "FEATURE_FLAG"
         ):
             return True
         return False
 
-    def resolve_context_argument(self, context):
+    def resolve_context_argument(self, context: str | Context) -> Context:
         if context != "NO_CONTEXT_PROVIDED":
             return context
         return Context.get_current()
 
-    def context(self):
+    def context(self) -> Context:
         return Context.get_current()
 
-    def scoped_context(context):
+    def scoped_context(context: Context) -> ScopedContext:
         return Context.scope(context)
 
     @functools.cache
-    def config_client(self):
+    def config_client(self) -> ConfigClient:
         client = ConfigClient(self, timeout=5.0)
         return client
 
     @functools.cache
-    def feature_flag_client(self):
+    def feature_flag_client(self) -> FeatureFlagClient:
         return FeatureFlagClient(self)
 
-    def post(self, path, body):
+    def post(self, path: str, body: PostBodyType) -> None:
```

Comment:
we might not be doing anything with it now, but we should probably return the response from `self.session.post()`